resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/config.scala

```diff
@@ -27,233 +27,240 @@ package object config {
 
   private[spark] val KUBERNETES_NAMESPACE =
     ConfigBuilder("spark.kubernetes.namespace")
-      .doc("""
-          | The namespace that will be used for running the driver and
-          | executor pods. When using spark-submit in cluster mode,
-          | this can also be passed to spark-submit via the
-          | --kubernetes-namespace command line argument.
-        """.stripMargin)
+      .doc("The namespace that will be used for running the driver and executor pods. When using" +
```

Comment:
I reformatted all of these because it's difficult to get triple-quoting to have a consistent format.